app/controllers/catalog_controller.rb

```diff
@@ -144,10 +144,10 @@ def exclude_unwanted_records(solr_parameters,user_parameters)
     # solr fields to be displayed in the index (search results) view
     #   The ordering of the field names is the order of the display
 	#	UCSD custom added argument :hitsonly to display the values that has hit text.
+    config.add_index_field 'collection_name_tesim', :label => 'Collection:', :highlight => config.highlighting
```

Comment:
Hi Vivian, why was the field name changed from 'collection_1_name_tesim' to 'collection_name_tesim'?